### PR TITLE
Update OTP code content

### PR DESCRIPTION
**Pull request by David-Feetenby**
Update the content for OTP code so as to make it less confusing as defined  by https://docs.google.com/document/d/1jaS_nxLyoakHdXZrLA7XGDdNhqf1duVH1AepVxJHLkk/edit

### CODE REVIEW

**Comment by David-Feetenby** [comment]
[users confused by confirmation code (Dev ticket)](https://trello.com/c/OZyQ8XXF/1529-users-confused-by-confirmation-code-dev-ticket)

File: app/views/teachers/mailer/otp.text.erb [hunk 1]
```diff
@@ -1,4 +1,6 @@
-Use this code to confirm your email address:
+<b> This is an automated email, please do not reply. </b>
```
Comment (thomasleese) [inline app/views/teachers/mailer/otp.text.erb:1]: We can't use HTML tags in email formatting as Notify doesn't support them - is there an alternative formatting we can use?

https://www.notifications.service.gov.uk/using-notify/guidance/edit-and-format-messages#formatting

File: app/views/teachers/otp/new.html.erb [hunk 2]
```diff
@@ -7,9 +7,11 @@
   <%= f.hidden_field :uuid, value: @otp_form.uuid %>
   <%= f.hidden_field :email, value: @otp_form.email %>
 
-  <p class="govuk-body">We’ve emailed a confirmation code to <%= @otp_form.email %>.</p>
+  <p class="govuk-body">We’ve sent a login code to your email address <%= @otp_form.email %>.</p>
+  <p class="govuk-body">Use the code to sign into the Apply for QTS in England Service</p>
+  <p class="govuk-body">The code will expire after 15 minutes.</p>
 
-  <%= f.govuk_text_field :otp, size: "m", label: { text: "Enter your code", class: "govuk-label--s" }, class: "govuk-!-width-one-quarter" %>
+  <%= f.govuk_text_field :otp, size: "m", label: { text: "Enter code:", class: "govuk-label--s" }, class: "govuk-!-width-one-quarter" %>
```
Comment (thomasleese) [inline app/views/teachers/otp/new.html.erb:14]: We don't use colons elsewhere in forms - is this something we definitely want?

Comment (David-Feetenby) [reply app/views/teachers/otp/new.html.erb:14]: It is what is in the content changes, but i can double check with dom

File: app/views/teachers/mailer/otp.text.erb (hunk 1 above)
Comment (David-Feetenby) [reply app/views/teachers/mailer/otp.text.erb:1]: I'll check with dom. Perhaps it can be a heading in stead of bold

**Review by thomasleese — APPROVED** [review]

**Pull request merged**